payments-core/src/main/java/com/stripe/android/payments/bankaccount/CollectBankAccountLauncher.kt

```diff
@@ -122,6 +129,21 @@ interface CollectBankAccountLauncher {
                 }
             )
         }
+
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun create(
```

Comment:
If we add the lifecycleOwner here, we could use the other register method that takes the lifecycleOwner, I think that one unregisters by itself. I would like to avoid making callers of this have to remember to unregister.

Reply:
`LifecycleOwner` doesn’t provide access to the `activityResultRegistry`, but I can add an internal interface `CollectBankAccountLauncherHost : LifecycleOwner, ActivityResultRegistryOwner` that we can hand to `create()` instead. I’ll push a commit with that change.

Reply:
Nevermind, I’ll simply use the current activity instead.